cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/util/ConfigurationFinder.java

```diff
@@ -101,14 +101,25 @@ public static URI findTemplatesLocation() {
 
       // use new template set configuration
       Path templateSetsFolderLocation = getTemplatesFolderLocation(cobigenHome, configFile, templateSetsLocation);
-      if (templateSetsFolderLocation != null && Files.exists(templateSetsFolderLocation)) {
-        return templateSetsFolderLocation.toUri();
+      if (templateSetsFolderLocation != null) {
+        if (!Files.exists(templateSetsFolderLocation)) {
+          LOG.warn(
+              "The value of the template-set path from the .cobigen file, does not exist. Using home directory:{} to search for templates",
+              cobigenHome);
+        } else {
+          return templateSetsFolderLocation.toUri();
+        }
       }
-
       // use old templates configuration
       Path templatesFolderLocation = getTemplatesFolderLocation(cobigenHome, configFile, templatesLocation);
-      if (templatesFolderLocation != null && Files.exists(templatesFolderLocation)) {
-        return templatesFolderLocation.toUri();
+      if (templatesFolderLocation != null) {
+        if (!Files.exists(templatesFolderLocation)) {
+          LOG.warn(
+              "The value of the templates path from the .cobigen file, does not exist. Using home directory:{} to search for templates",
```

Comment:
Text part: "path from the .cobigen file, does not exist. Using home directory:{} to search for" should be moved into a constant.

Reply:
Removed the Logs because this will already be logged in GetTemplatesFolderLocation.